sklearn-demo.ipynb

```diff
@@ -16,7 +16,7 @@
   },
```

Comment:
<p>this is a comment</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/maxjakob/reviewnb-playground/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>replying to a comment</p>